doc/source/installation.rst

```diff
@@ -18,26 +18,19 @@ The stable version is tried and tested, and it normally operates on a stable
 version of yt.  The development version is actively being updated with new
 features, and it is also tied to the development version of yt, so occasionally
 unforseen bugs can crop up as these new features are added.  The demeshening
```

Comment:
The demeshening terminology seems to have fallen away. It might be clearer to just refer to this as `yt-4.0` as opposed to `yt-3.x`.

Reply:
Addressed.